core-js version conflict

# Expected Behavior

For jimp not to crasch when required.

# Current Behavior

Crash on require.

# Failure Information (for bugs)

Stack trace:

Error: Cannot find module 'core-js/modules/es6.object.define-property'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/srv/node_modules/jimp/dist/index.js:3:1)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)

## Steps to Reproduce
1. Add dependency to core-js 1.2.7
2. Add dependency to latest jimp
3. Require jimp

## Context

- Jimp Version: 0.3.2
- Operating System: Ubuntu 18.04
- Node version: 9.11.2

--- a/packages/jimp/package.json
+++ b/packages/jimp/package.json
@@ -57,7 +57,8 @@
     "@babel/polyfill": "^7.0.0-rc.1",
     "@jimp/custom": "^0.3.7",
     "@jimp/plugins": "^0.3.7",
-    "@jimp/types": "^0.3.7"
+    "@jimp/types": "^0.3.7",
+    "core-js": "^2.5.7"
   },
   "devDependencies": {
     "babelify": "^9.0.0",

